lib/kubeclient/common.rb

```diff
@@ -115,9 +115,9 @@ def get_entities(entity_type, klass, options)
           params['params'] = { labelSelector: options[:label_selector] }
         end
 
-        # TODO: namespace support?
+        ns_prefix = build_namespace_prefix(options[:namespace])
```

Comment:
FYI - in Kubernetes there are label selectors to filter resources & field selectors to specify what fields to return.